README.md

```diff
@@ -74,17 +76,26 @@ Next, update your `AppDelegate` to include the following function to allow the r
 import OktaAuth
 
 func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any]) -> Bool {
-  return OktaAuth.resume(url: url, options: options)
+  // oktaAuth - Configured OktaAppAuth instance used to start SignIn/SignOut flow. 
+  return oktaAuth.resume(url: url, options: options)
```

Comment:
Developers will also need the following line for this method to work:

```swift
var oktaAuth: OktaAppAuth?
```

Reply:
Fixed.